recipes/knit/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.2.2" %}
+{% set name = "knit" %}
+
+package:
+    name: {{name}}
+    version: {{version}}
+
+source:
+  fn: {{name}}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/k/{{name}}/{{name}}-{{ version }}.tar.gz
+  sha256: 3d847be729bd8f25bca7da993ecea44600ac31b90a553e03c14da575a0add004
+  
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - maven
+        - openjdk
+    run:
+        - python
+        - lxml
+        - requests
+        - py4j
+
+test:
+  imports:
+    - knit
+
+about:
+  home: https://github.com/dask/knit
+  license: BSD 3-Clause
+  license-family: BSD
+  license_file: LICENSE
+  summary: Python interface YARN
+
+extra:
+  recipe-maintainers:
+    - quasiben 
```

Comment:
Are you ok being a maintainer here, @quasiben?

Reply:
Yeah, i'm ok with it.  Apologies for the delay

Reply:
No worries. Thanks.